hypershift-operator/controllers/hostedcluster/utils/utils_test.go

```diff
@@ -0,0 +1,68 @@
+package utils
```

Comment:
Done, I've moved all the metrics recollection which contains logic. I didn't modify my metrics because it does not contains any logic to be collected. Do you think that makes sense to add their own function or just keep it as it is?